crates/bevy_wgpu/src/wgpu_renderer.rs

```diff
@@ -19,9 +19,15 @@ pub struct WgpuRenderer {
 impl WgpuRenderer {
     pub async fn new() -> Self {
         let instance = wgpu::Instance::new(wgpu::BackendBit::PRIMARY);
+    
+        #[cfg(feature = "low_power")]
+        let power_preference = wgpu::PowerPreference::Default;
```

Comment:
Naming a feature `low_power` and then mapping it to `PowerPreference::Default` instead of `PowerPreference::LowPower` may be confusing to users (it certainly would be to me).

Suggestion:
- If you intend to _force_ things into low power mode, then map the feature to `PowerPreference::LowPower` 
- If you want to switch to the more powerful GPU when main power is connected (which is what `PowerPreference::Default` does), then rename the feature so it more accurately describes what it does.  Maybe `adaptive_power` or something like that?

Reply:
Changed the option to "adaptive_power", added the "low_power" feature as PowerPreference::LowPower

Reply:
> If you want to switch to the more powerful GPU when main power is connected (which is what PowerPreference::Default does)

I don't think this is true any longer. [The `power` module was removed.](https://github.com/gfx-rs/wgpu/pull/864) I believe `PowerPreference::Default` now always prefers an integrated GPU over discrete when both are available. This behavior is odd as nearly every other software behaves in the opposite way.